src/main/java/com/searchclient/clientwrapper/domain/service/InputDocumentService.java

```diff
@@ -77,7 +77,7 @@ private JSONObject uploadData(String payload,String url, String jwtToken) {
     private void generateResponse(IngestionResponse ingestionResponseDTO, String message) {
         log.debug(message);
         ingestionResponseDTO.setMessage(message);
-        ingestionResponseDTO.setStatusCode(400);
+        ingestionResponseDTO.setStatusCode(108);
     }
 
     private String verify(String payload, Map<String, Map<String, Object>> schemaKeyValuePair) {
```

Comment:
What kind of verification is performed here? method name doesn't suggest anything..